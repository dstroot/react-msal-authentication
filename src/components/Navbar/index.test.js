import React from "react";
import renderer from "react-test-renderer";
import { BrowserRouter as Router } from "react-router-dom";
import NavBar from "../NavBar";

describe("NavBar", () => {
  it("it should render", () => {
    const component = renderer.create(
      <Router>
        <NavBar />
      </Router>
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
