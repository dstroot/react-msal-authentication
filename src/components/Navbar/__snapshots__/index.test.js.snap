// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar it should render 1`] = `
<nav
  className="navbar fixed-top navbar-expand-sm navbar-light bg-light"
>
  <div
    className="container"
  >
    <a
      aria-current="page"
      className="navbar-brand"
      href="/"
      onClick={[Function]}
    >
      <img
        alt="Pacific Life"
        className="d-inline-block align-middle"
        height="40"
        src="logo-horiz.svg"
        width="220"
      />
    </a>
    <button
      aria-label="Toggle navigation"
      className="navbar-toggler collapsed"
      onClick={[Function]}
      type="button"
    >
      <span
        className="navbar-toggler-icon"
      />
    </button>
    <div
      className="navbar-collapse collapse"
    >
      <a
        aria-current="page"
        className="active nav-link"
        href="/"
        onClick={[Function]}
      >
        Home
      </a>
      <a
        className="nav-link"
        href="/about"
        onClick={[Function]}
      >
        About
      </a>
      <a
        className="nav-link"
        href="/news"
        onClick={[Function]}
      >
        News
      </a>
    </div>
  </div>
</nav>
`;
